src/ape/managers/converters.py

```diff
@@ -140,6 +141,27 @@ def convert(self, value: Union[List, Tuple]) -> Union[List, Tuple]:
 list_tuple_converter = ListTupleConverter(None, None, None)  # type: ignore
 
 
+class TimestampConverter(ConverterAPI):
+    """Converts seconds to date and time string value"""
```

Comment:
I think we should include the string format(s) required in the doc here so users know what is possible.